x-pack/plugins/cross_cluster_replication/server/lib/__snapshots__/follower_index_serialization.test.js.snap

```diff
@@ -96,3 +96,20 @@ Object {
   "writeBufferSizeBytes": "write buffer size in bytes",
 }
 `;
+
+exports[`[CCR] follower index serialization serializeFollowerIndex() serializes object to Elasticsearch follower index object 1`] = `
+Object {
```

Comment:
Can we avoid using snapshots for non-rendered assertions? It's easier to read the test when the object is inlined in the assertion, and I doubt this will be changing often enough to make the snapshot-generation a timesaver during dev.